project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.views_default.inc

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * @file
+ * c4m_features_homepage.views_default.inc
+ */
+
+/**
+ * Implements hook_views_default_views().
+ */
+function c4m_features_homepage_views_default_views() {
+  $export = array();
+
+  $view = new view();
+  $view->name = 'overview_og_related_projects';
+  $view->description = '';
+  $view->tag = 'default';
+  $view->base_table = 'node';
+  $view->human_name = 'Overview - OG - Related Projects';
+  $view->core = 7;
+  $view->api_version = '3.0';
+  $view->disabled = FALSE; /* Edit this to true to make a default view disabled initially */
+
+  /* Display: Master */
+  $handler = $view->new_display('default', 'Master', 'default');
+  $handler->display->display_options['title'] = 'Related Projects';
+  $handler->display->display_options['use_more_always'] = FALSE;
+  $handler->display->display_options['access']['type'] = 'perm';
+  $handler->display->display_options['cache']['type'] = 'none';
+  $handler->display->display_options['query']['type'] = 'views_query';
+  $handler->display->display_options['exposed_form']['type'] = 'basic';
+  $handler->display->display_options['pager']['type'] = 'some';
+  $handler->display->display_options['pager']['options']['items_per_page'] = '3';
+  $handler->display->display_options['style_plugin'] = 'default';
+  $handler->display->display_options['row_plugin'] = 'fields';
```

Comment:
@guytepper There is if you take "Content" as style. "Block List" is available as view mode then.